.gitignore

```
@@ -0,0 +1,41 @@
+_site
```

t'es sûre qu'il faut pas mettre ?
```suggestion
docs/_site
```

---

J'ai testé comme ça, je n'ai pas eu les fichiers dans _site quand j'ai fait `git status`

---

Du coup, j'en ai conclu que ce n'était pas obligatoire de rajouter `docs/`
si tu peux tester de builder et voir si ça génère `_site` et si ça te le proposes comme changements à commit ?

---

ok, je sais jamais. Il me semble qu'effectivement ça marche avec une partie du path. Attend je teste.

---

ca marche :tada: